earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -297,21 +298,23 @@ determine_os()
     local test_versionid=""
 
     if [ -f "$OS_RELEASE1" ] || [ -f "$OS_RELEASE2" ]; then
-		if [ -f "$OS_RELEASE1" ]; then
+	if [ -f "$OS_RELEASE1" ]; then
 	        test_os="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^NAME= | sed 's:name=::gI')"
-    	    test_versionid="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^VERSION_ID= | sed 's:version_id=::gI')"
-		else
-			test_os="$(cat $OS_RELEASE2 | sed 's:[0-9\.] *::g')"
-			test_versionid="$(cat $OS_RELEASE2 | sed 's:[^0-9\.]*::g')"
-		fi
+    		test_versionid="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^VERSION_ID= | sed 's:version_id=::gI')"
```

Comment:
Let's talk about that -- I'll admit nano is not giving me consistent results at the moment on the tabs, but no tabs?  That's an age old debate -- spaces vs. tabs!  :)